src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -24,23 +24,18 @@ public class DeleteCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1";
```

Comment:
Hi, since the implementation of delete command is by ic number, would it be good to also update the message usage accordingly?

Reply:
Done!